src/data/queries/personProfile.ts

```diff
@@ -45,19 +44,7 @@ export const data: QueryData<PersonProfileDataOpts, PersonProfileData> = async (
     params
   )) as NodePersonProfile
 
-  // Fetch the menu name dynamically off of the field_office reference if available.
-  let menu = null
-  if (entity.field_office?.field_system_menu) {
-    menu = await getMenu(
-      entity.field_office.field_system_menu.resourceIdObjMeta
-        .drupal_internal__target_id
-    )
-  }
-
-  return {
-    entity,
-    menu,
-  }
```

Comment:
This menu bit seems to be vestigial. I asked Laura about if the sidebar ever shows more than a link back to the listing page and she said no.